low-level/files/opt/seagate/sspl/bin/generate_sspl_bundle/sspl_bundle_generate

```diff
@@ -325,21 +324,35 @@ def sspl_bundle_cleanup():
     if os.path.exists(sspl_bundle_tmp):
         shutil.rmtree(sspl_bundle_tmp)
 
+def validate_sb_request():
+    """Checking previous process pid/request is running or not."""
+    valid_request = True
+    sb_pid = None
+    if os.path.isfile(onGoingRequestPath):
+        with open(onGoingRequestPath,'r') as f:
+            sb_pid = f.read()
+        if sb_pid and psutil.pid_exists(int(sb_pid)):
+            valid_request = False
+        else:
+            sspl_bundle_cleanup()
+    return valid_request
+
 if __name__ == "__main__":
-    if not os.path.isfile(onGoingRequestPath):
-        if len(sys.argv) < 3:
-            print("Unrecognized arguments: %s" % sys.argv)
-            sys.exit(1)
-        tmpFile = open(onGoingRequestPath, "w")
+    if len(sys.argv) < 3:
+        print("Unrecognized arguments: %s" % sys.argv)
+        sys.exit(1)
+    valid_request = validate_sb_request()
+    if valid_request:
+        with open(onGoingRequestPath,'w') as f:
+            f.write(str(os.getpid()))
```

Comment:
In else, we should fail with error i.e. sys.exit(1)

Reply:
I have covered this in else part of 358.

Reply:
Done